dev-tools/packaging/packages.yml

```diff
@@ -366,6 +366,10 @@ specs:
       spec:
         <<: *macos_beat_pkg_spec
         <<: *elastic_license_for_macos_pkg
+        files:
+          /Library/Application Support/{{.BeatVendor}}/{{.BeatName}}/bin/{{.BeatName}}{{.BinaryExt}}:
+            mode: 0755
```

Comment:
It's weird that this `mode: 0755` is needed. Without it, the binary in the tree that is passed to pkgutil has all-zero permissions and generation of the .pkg fails.